lib/SPIRV/libSPIRV/SPIRVDebug.cpp

```diff
@@ -45,3 +45,8 @@ bool SPIRV::SPIRVDbgEnable = false;
 SPIRV::SPIRVDbgErrorHandlingKinds SPIRV::SPIRVDbgError =
     SPIRVDbgErrorHandlingKinds::Exit;
 bool SPIRV::SPIRVDbgErrorMsgIncludesSourceInfo = true;
+
+llvm::cl::opt<bool> SPIRV::VerifyRegularizationPasses(
+    "spirv-verify-regularize-passes", llvm::cl::init(_SPIRVDBG),
+    llvm::cl::desc(
+        "Validate module after each pass in LLVM regularization phase"));
```

Comment:
```suggestion
        "Verify module after each pass in LLVM regularization phase"));
```

Reply:
Applied in 33c039b949569438afeba506ef8dfa7c56ed2a17